packages/graphback-runtime-knex/tests/data/KnexDbDataProviderTest.ts

```diff
@@ -4,19 +4,26 @@
 import { unlinkSync, existsSync } from 'fs';
 import { buildSchema } from 'graphql';
 import * as Knex from 'knex';
-import { GraphbackDataProvider, GraphbackCoreMetadata } from '@graphback/core';
+import { GraphbackDataProvider, GraphbackCoreMetadata, QueryFilter } from '@graphback/core';
 import { SQLiteKnexDBDataProvider } from '../../src/SQLiteKnexDBDataProvider';
 import { migrateDB, removeNonSafeOperationsFilter } from '../../../graphql-migrations/src';
 
 const dbPath = `${__dirname}/db.sqlite`;
 
+beforeEach(() => {
+  if (existsSync(dbPath)) {
+    unlinkSync(dbPath)
+  }
+})
+
 afterEach(() => {
   if (existsSync(dbPath)) {
     unlinkSync(dbPath)
   }
 })
```

Comment:
I suppose this is needed because we are not using the in `memory` connection? Was there any preference of using `file system` over an in memory one?

Reply:
Actually I put the second one in as a temporary thing because I was running the tests in parallel in two places (watch mode and through VS Code and they were conflicting), I can take this out again.

And there probably was some reason at the time, but I feel like it has been obsoleted and can change it now.

Reply:
Running into issues: `SQLITE_ERROR: no such table: todo` on ever test case. I'll leave it as is for now.

Reply:
Okay, I just spotted it. As you prefer, but if we can use an in memory store maybe the test would be somehow faster and can be ran in parrallel?